server/server.js

```diff
@@ -6,6 +6,6 @@ const app = express();
 app.use(express.static('client/dist'));
 app.use(express.json());
 
-app.listen(process.env.PORT, () => {
-  console.log('Client server listening on 3000');
+app.listen(6000, () => {
```

Comment:
This change seems arbitrary. How does it relate to docker changes or why is it necessary?

Reply:
ok, do a new commit with it as 3000